x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -15,301 +17,281 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
   {
     key: 'linux.advanced.agent.connection_delay',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: i18n.translate(
```

Comment:
@kevinlog can you confirm that this looks right before i add it for each key?

Reply:
looks good!

Reply:
First supported version is `7.9`.

Reply:
fixed both in ffa65db31caa5a5bca2ad19868587d115a8ffe04